CHANGELOG.next.asciidoc

```diff
@@ -479,6 +479,9 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add `elasticsearch/enrich` metricset. {pull}14243[14243] {issue}14221[14221]
 - Add support for Application ELB and Network ELB. {pull}14123[14123] {issue}13538[13538] {issue}13539[13539]
 - Release aws ebs metricset as GA. {pull}14312[14312] {issue}14060[14060]
+- Add `connection.state` field for RabbitMQ module. {pull}13981[13981]
```

Comment:
oops this need to be removed?

Reply:
yeap, thnak you for catching this @kaiyan-sheng